docker/Dockerfile.base

```diff
@@ -0,0 +1,11 @@
+FROM nvidia/cuda:11.4.0-cudnn8-runtime-ubuntu20.04
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get update -yy \ 
+    && apt-get install -yy --no-install-recommends python3 python3-pip ffmpeg libsm6 libxext6 \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN pip install --no-cache-dir deeplabcut>=2.2.0.2
```

Comment:
RUN pip install --no-cache-dir deeplabcut[gui]>=2.2.0.2 would install the full GUI; then light needs to be just RUN pip install --no-cache-dir deeplabcut>=2.2.0.2

Reply:
This is resolved now I think. `base` will installed the standard DLC package, `gui` will add the GUI package, but with updated versions.